PWGEM/PhotonMeson/Tasks/emcalQC.cxx

```diff
@@ -156,16 +160,16 @@ struct emcalQC {
 
   void processQC(MyCollisions const& collisions, MyEMCClusters const& clusters)
   {
-    for (auto& collision : collisions) {
+    for (const auto& collision : collisions) {
 
-      if (eventcuts.onlyKeepWeightedEvents && fabs(collision.weight() - 1.) < 1E-10) {
+      if (eventcuts.onlyKeepWeightedEvents && std::fabs(collision.weight() - 1.) < 1E-10) {
         continue;
       }
 
       fRegistry.fill(HIST("Event/hEMCCollisionCounter"), 1);
       if (collision.selection_bit(o2::aod::evsel::kIsTriggerTVX)) {
         fRegistry.fill(HIST("Event/hEMCCollisionCounter"), 2);
-        if (abs(collision.posZ()) < eventcuts.cfgZvtxMax) {
+        if (collision.posZ() < eventcuts.cfgZvtxMax && collision.posZ() > -eventcuts.cfgZvtxMax) {
```

Comment:
Why did you remove the `abs`?

Reply:
I got a warning about the abs by itself and then checked out how it was done in the PWGEM/Dilepton/Core/EMEventCut.h and wanted to make it more similar to the way it was implemented there

Reply:
But `EMEventCut` has `mMinZvtx` and `mMaxZvtx`. You only have `cfgZvtxMax`.

Reply:
Yes, however, I don't plan on ever using an asymmetric z vertex cut.
I don't really understand the point of this change request. Do you not like the style of my if statement, or would you prefer EMCal QC to be done in an asymmetric z vertex region, or what is blocking this PR?

Reply:
I understood the subject of this PR was to fix the abs bug. What you did instead is to change the condition to support asymmetric selection without actually using it.

Reply:
Sorry, maybe I don't fully understand: Does this not implicitly solve the issue?

Reply:
It's not clear to me why you would prefer
```c++
collision.posZ() < eventcuts.cfgZvtxMax && collision.posZ() > -eventcuts.cfgZvtxMax
```
instead of
```c++
std::abs(collision.posZ()) < eventcuts.cfgZvtxMax
```
if don't use and will never use an asymmetric cut.

Reply:
To be honest, I do not have a preference at all. But I am happy to comply with the request

Reply:
Thanks